simulation/setup.py

```diff
@@ -1,43 +1,42 @@
 from distutils.core import setup
-
 from setuptools import find_packages
 
 # When publishing the Docker image, a script checks for the first line with "version" and an equals sign to get the version.
 version='1.0.0'
 
+# List of packages required by the application
 install_requires = [
-    'bokeh>=0.13',
-    'expiringdict>=1.1.4',
-    'injector>=0.16.2',
-    'joblib>=0.13.2',
-    'keras>=2.3',
-    'mmh3~=3.0.0',
-    'numpy',
-
-    # Required for saving plots.
-    'selenium>=3.141.0',
-    'scikit-multiflow>=0.3.0',
-    'spacy>=2.2',
-
-    'tqdm>=4.19',
+    'bokeh>=0.13',              # Interactive visualization library
```

Comment:
I know people like comments at the end of a line, but I find them really distracting. Comments should come before code to explain WHY the code is doing something. In this case, putting the comments on the same line also make the diff look scary and I need to manually inspect each line to see if the string has changed.